### PR TITLE
Add bazel_sonarqube

### PR DESCRIPTION
**Pull request by adincebic**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (bazel_sonarqube) have been updated in this PR. Please review the changes.

File: modules/bazel_sonarqube/1.0.1/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  bazel:
+  - 7.x
+  - 6.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@bazel_sonarqubesonarqube'
```
Comment (fmeum) [inline modules/bazel_sonarqube/1.0.1/presubmit.yml:17]: This looks like a typo

Comment (adincebic) [reply modules/bazel_sonarqube/1.0.1/presubmit.yml:17]: Oh thanks for catching that. I will need to update the original repo first to add a target to test.

Comment (adincebic) [reply modules/bazel_sonarqube/1.0.1/presubmit.yml]: @fmeum I believe this is now ready to go.

**Review by fmeum — APPROVED** [review]

File: modules/bazel_sonarqube/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/Zetten/bazel-sonarqube",
+    "maintainers": [
+        {
+            "email": "peter.vanzetten@gmail.com",
+            "github": "zetten",
```
Comment (fmeum) [inline modules/bazel_sonarqube/metadata.json:6]: @zetten Do you agree with this submission?

@adincebic Could you also add yourself here? This field is not just meant for upstream maintainers, but for those who handle BCR submissions for this module.

**Review by fmeum — APPROVED** [review]

File: modules/bazel_sonarqube/metadata.json (hunk 2 above)
Comment (adincebic) [reply modules/bazel_sonarqube/metadata.json:6]: Done.

Comment (Zetten) [reply modules/bazel_sonarqube/metadata.json:6]: > @Zetten Do you agree with this submission?

Yes,  and many thanks @adincebic!

**Review by fmeum — APPROVED** [review]

**Pull request merged**